lib/matplotlib/tests/test_axes.py

```diff
@@ -1695,6 +1695,14 @@ def test_bar_pandas_indexed(pd):
     ax.bar(df.x, 1., width=df.width)
 
 
+def test_pandas_smoke(pd):
```

Comment:
Smoke tests are simple tests to verify basic functionality, without checking details, such as „does it run without producing errors“. Not sure why it’s called that way. Maybe because the program smokes metaphorically if it’s not working properly or blows up?

https://en.wikipedia.org/wiki/Smoke_testing_(software)

Reply:
My understanding of the history of that phrase is that with physical hardware if it fails the test it starts smoking.  Which leads to the joke that "electronics run on smoke" (like steam engines run on steam) and when it breaks "you let the smoke out".

Reply:
Ok, then can we change the name of this function to something that's not idiomatic like `test_pandas_empty` or `test_accepts_pandas`?